src/App.jsx

```diff
@@ -1,13 +1,21 @@
+import { BrowserRouter as Router, Switch, Route, Link } from 'react-router-dom'
 import BookList from './views/Books/BookList'
 import BookDetail from './views/Books/BookDetail'
+import Home from './views/Books/Home'
 import './App.css'
 
 function App() {
   // TODO: Add routes to books & views
   return (
     <main className="container">
-      <h1>Library Catalog</h1>
-      <BookList />
+      <Router>
```

Comment:
You can also add a header either here or as a separate component to always show the NavLinks to the Home and the Catalog screens